### DISCUSSION TITLE
Please add nowpayments

### DISCUSSION THREAD

**Discussion by LaffDev** [Paymenter/Paymenter#478] [Ideas]
https://nowpayments.io/

**Comment by GomorrhaDev** [comment]
Duplicate of https://github.com/Paymenter/Paymenter/discussions/194

**Comment by GH0st3rs** [comment]
Hi You can try to use my integration.
https://github.com/GH0st3rs/NOWpayments-paymenter

Fill free to submit Issue and PR
